routing_common/vehicle_model.hpp

```diff
@@ -55,6 +55,7 @@ enum class HighwayType : uint16_t
   HighwaySecondaryLink = 176,
   RouteFerry = 259,
   HighwayTertiaryLink = 272,
+  HighwayServiceEmergency = 350,
```

Comment:
Well, there is a bit tricky here. Adding a separate 3-arity type into routing algo (highway-service-emergency) won't work as expected for now, unfortunately.